greedy_methods/interval_scheduling.py

```diff
@@ -0,0 +1,27 @@
+# Interval Scheduling: https://en.wikipedia.org/wiki/Interval_scheduling
```

Comment:
Also, please provide an explanation of the algorithm: what problem it solves (interval scheduling, in this case), how it works, its time complexity, etc.